lms/djangoapps/mobile_api/course_info/tests.py

```diff
@@ -4,15 +4,13 @@
 
 import ddt
 from django.conf import settings
-
 from xmodule.html_module import CourseInfoModule
 from xmodule.modulestore import ModuleStoreEnum
 from xmodule.modulestore.django import modulestore
 from xmodule.modulestore.xml_importer import import_course_from_xml
-
 from milestones.tests.utils import MilestonesTestCaseMixin
 
-from ..testutils import (
+from mobile_api.tests.testutils import (
```

Comment:
Nit: Don't need this change.  Explicit relative imports are allowed and even recommended by the 2 scoops django book.  What's not allowed are "implicit" relative imports that aren't preceded by `.`.